### DISCUSSION TITLE
request menu

### DISCUSSION THREAD

**Discussion by doko89** [hotspotbilling/phpnuxbill#51] [Ideas]
mas boleh request menu toggle enable disable register ga, soalnya untuk halaman public kawatirnya ada yang spam di register

**Comment by ibnux** [comment]
Akan dipertimbangkan

makanya ada registrasi menggunakan nomor HP, untuk mengurangi spam

**Comment by webwhisker** [comment]
Hi sir phpbill will work on Static client? how to add services dhcp/static for client

**Reply by ibnux** [reply]
Too bad it can't

Phpnuxbill depend on address pool

**Comment by fernandus43** [comment]
om boleh request tambahkan menu dedicated seperti simple queue di mikrotik. biar selain pppoe bisa dimanage via phpnuxbill.
